frontend/src/components/partners/partnersActivity.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import ReactPlaceholder from 'react-placeholder';
+import PartnersProgresBar from './partnersProgresBar';
+import messages from '../teamsAndOrgs/messages';
+import { OHSOME_STATS_BASE_URL } from '../../config';
+import { FormattedMessage } from 'react-intl';
+export const Activity = () => {
+  const [data, setData] = useState(null);
+
+  const fetchData = async () => {
+    try {
+      const response = await fetch(
+        OHSOME_STATS_BASE_URL +
+          '/hashtags/accenture,acn*,dublinacn19,acngraddublin19,acnfy22,acnfy23',
```

Comment:
These hashtags should not be hardcoded

Reply:
Hi, I’ve been testing and I’m not sure which endpoint to use to get all the hashtags. We used the example you provided. Let me know what you want me to do there, and I'll change it.
this is the example URL:
https://stats.now.ohsome.org/api/stats/hashtags/accenture,acn*,dublinacn19,acngraddublin19,acnfy22,acnfy23

Reply:
The URL has hashtags for one partner (accenture), you should use the hashtags assigned to the partner in the admin form:

`https://stats.now.ohsome.org/api/stats/hashtags/<primary hashtag>,<secondary hashtag>`

<img width="495" alt="Screenshot 2024-06-06 at 15 31 56" src="https://github.com/hotosm/tasking-manager/assets/1226194/3a9cdfc1-71ab-4a50-b75b-019b9f170259">

CC @ramyaragupathy

Reply:
We used the endpoint to call all the partners for comparison, just like in the example, and with that data, we created these three graphics:

![image](https://github.com/hotosm/tasking-manager/assets/72893031/3e9a3950-6b16-46c8-b389-fd03e6d8c368)

and we need to call all the hashtags there.

Reply:
Sorry if this was not clear, but the statistics are based on the hashtags that are assigned to the Partner.

Here's the specification in the [repository](https://github.com/MissingMaps/partners) that builds the current [Leaderboard page](https://www.missingmaps.org/partners/accenture/):

| Field         | Changes  |
| ------------- |:-------------:|
| primary-hashtag | The overall hashtag for the partner. Used to populate the primary stats at the top of the page. |
| subhashtags | Subhashtags that create the 'team' section. Must be in the format set in the example partner.

So, primaryHashtag is the overall hashtag for the partner and it's used for the primary stats:

<img width="620" alt="Screenshot 2024-06-06 at 17 28 25" src="https://github.com/hotosm/tasking-manager/assets/1226194/ccdaf42b-e6db-48dc-ade9-cbcaea85d3ce">

While in the secondary field the user can use several hashtags and those are used for the _team_ section, that's what you have there under the "Actividad" title.

CC @ramyaragupathy

Reply:
Another examples of partner's pages are:

https://www.missingmaps.org/partners/msf 
https://www.missingmaps.org/partners/aws